drivers/input/Kconfig

```diff
@@ -0,0 +1,32 @@
+# Input subsystem configuration options
+
+# Copyright (c) 2022, tangchunhui@coros.com
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig INPUT
+	bool "Input Subsystem Drivers"
+	select RING_BUFFER
+	help
+	  Include input subsystem drivers in system config.
+
+if INPUT
+
+config ENABLE_INPUT_ISR_LOCK
+	bool "Enable input isr lock"
+	help
+	  Use isr_lock() and isr_unlock() to control ring buffer exclusive access.
+	  This is use to report input event in isr context.
```

Comment:
The description is quite confusing. "This is required to be able to report input event in isr context."  might be better.